Makefile

```diff
@@ -0,0 +1,250 @@
+# -*- Mode: makefile-gmake -*-
```

Comment:
Are these makefiles generated? Would it be better to do that on build rather than check these in?

Reply:
That just sets the right editing mode for emacs